api/validation/IndicatorSchema.py

```diff
@@ -510,6 +510,12 @@ class DateValidationResponseSchema(BaseModel):
         examples=[True, False],
     )
 
+class MetadataOpts(BaseModel):
+    match_score: float
```

Comment:
Decided that any searches, either keyword or semantic, can and should include max_score (top-level results) and match_score (per result) properties